ghzdac/correction.py

```diff
@@ -1110,6 +1130,9 @@ def DACifyFT(self, signal, t0=0, n=8192, offset=0, nfft=None, loop=False,
         decayRates = np.array([x['GHz'] for x in self.decayRates])
         decayAmplitudes = self.decayAmplitudes
 
+        reflectionRates = np.array([x['GHz'] for x in self.reflectionRates])
```

Comment:
Yes, but all the variables inside the server (time, frequency etc) don't have labrad units. Internally, time is in ns, freq in GHz, I don't want to upset that right now